### PR TITLE
fix: catch exception when parsing file for transcripts

### PR DESCRIPTION
**Pull request by dixidroid**
try/catch block was missed for the second case when transcriptInputStream == null

### CODE REVIEW

File: core/src/main/java/org/openedx/core/module/TranscriptManager.kt [hunk 1]
```diff
@@ -67,9 +67,11 @@ class TranscriptManager(
             )
             if (result) {
                 getInputStream(downloadLink)?.let {
-                    val transcriptTimedTextObject =
-                        convertIntoTimedTextObject(it)
-                    transcriptObject = transcriptTimedTextObject
+                    try {
+                        transcriptObject = convertIntoTimedTextObject(it)
+                    } catch (e: Exception) {
```
Comment (farhan-arshad-dev) [inline core/src/main/java/org/openedx/core/module/TranscriptManager.kt:72]: [Optional] UI/UX Improvement
Space underneath the video player if the app is unable to load the transcript.

image

**Review by farhan-arshad-dev — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/openedx/core/module/TranscriptManager.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (dixidroid) [reply core/src/main/java/org/openedx/core/module/TranscriptManager.kt]: done

File: core/src/main/java/org/openedx/core/module/TranscriptManager.kt [hunk 2]
```diff
@@ -67,9 +67,11 @@ class TranscriptManager(
             )
             if (result) {
                 getInputStream(downloadLink)?.let {
```
[the remark this answers is not in the document]
Comment (farhan-arshad-dev) [reply core/src/main/java/org/openedx/core/module/TranscriptManager.kt:72]: As discussed, we will cover this in a separate PR.

File: core/src/main/java/org/openedx/core/module/TranscriptManager.kt (hunk 1 above)
Comment (farhan-arshad-dev) [reply core/src/main/java/org/openedx/core/module/TranscriptManager.kt]: As discussed, we will cover this in a separate PR.

**Review by farhan-arshad-dev — APPROVED** [review]

**Pull request merged**